VTEX - Marketplace Protocol - External Marketplace Orders.json

```diff
@@ -128,7 +128,7 @@
                                     },
                                     "isCreatedAsync": {
                                         "type": "boolean",
-                                        "description": "Indicates whether an order is created. It must be `true` if an order is being placed with [Price divergence](https://help.vtex.com/en/tutorial/price-divergence-rule--6RlFLhD1rIRRshl83KnCjW#), otherwise the request will not work.",
+                                        "description": "Indicates whether an order is created. It must be `true` if an order is being placed with [Price divergence](https://help.vtex.com/en/tutorial/configuracao-da-regra-de-divergencia-de-valores--awAKP0sS5J8jgLs2g7pPe), otherwise the request will not work.",
```

Comment:
```suggestion
                                        "description": "Indicates whether an order is created. It must be `true` if an order is being placed with [Price divergence](https://help.vtex.com/en/tutorial/configuring-price-divergence-rule--awAKP0sS5J8jgLs2g7pPe), otherwise the request will not work.",
```